src/renderers/WebGLRenderer.js

```diff
@@ -1486,6 +1486,8 @@ class WebGLRenderer {
 
 			const materialProperties = properties.get( material );
 
+			materialProperties.instancingColor = object.isInstancedMesh === true && object.instanceColor !== null;
```

Comment:
Do you mind explaining why this line required? I was under the impression the additional checks in `setProgram()` would be sufficient.

Reply:
It was left over. I verified that the code path was reached despite the bailout via `lightsStateVersion`, so this is redundant.